### PR TITLE
VIH-9827 Speed up user creation

### PR DESCRIPTION
**Pull request by oliver-scott**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-9827

### Change description ###
Fixes an issue where creating bookings with lots of new users can time out.

To remedy this, create the users in AAD in parallel, rather than sequentially.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/vh-booking-queue-subscriber/values.dev.template.yaml
Comment (oliver-scott) [inline charts/vh-booking-queue-subscriber/values.dev.template.yaml:1]: note: revert these changes before merging

**Review by shaed-parkar — DISMISSED** [review]
Looks good.
You wouldn't happen to have a bench mark on before and after do you?

**Review by shaed-parkar — COMMENTED** [review]
Undo the changes before you merge. missed the queue name

**Review by shaed-parkar — COMMENTED** [review]
Undo the changes before you merge. missed the queue name

**Comment by shaed-parkar** [comment]
> > Looks good. You wouldn't happen to have a bench mark on before and after do you?
> 
> Thanks :)
> 
> Before: 44 seconds to create 8 users After: < 1 second (each request runs in parallel, so total elapsed time equates to the longest response time)

That's incredible! Great job!!!

**Review by phildebrah — APPROVED** [review]
Tested and created 34 participants successful

**Review by will-craig — APPROVED** [review]

**Review by phildebrah — APPROVED** [review]
Reapprove
